### DISCUSSION TITLE
math 101

### DISCUSSION THREAD

**Discussion by TinyHust** [TinyHust/notes#418] [Math]
- [probability and statistics](https://github.com/TinyHust/notes/discussions/418#discussioncomment-10781687)

**Comment by TinyHust** [comment]
## probability and statistics
- [linear regression](https://github.com/TinyHust/notes/discussions/418#discussioncomment-10781701)

**Reply by mediumhust** [reply]
### linear regression
The **linear equation** used in regression is typically of the form:
$y = wx + b$

Where:
- **y** is the dependent variable
- **x** is the independent variable
- **w** is the slope of the line (how steep the line is)
- **b** is the y-intercept (where the line crosses the y-axis)

**Multiple Regression**
$y = b_0 + b_1 \times x_1 + b_2 \times x_2 + \dots + b_n \times x_n$

Where:
- $b_0$ is the intercept
- $b_1, b_2, \dots, b_n$ are the coefficients for each independent variable ($x_1, x_2, \dots, x_n$)
